content/en/docs/reference/scheduling/config.md

```diff
@@ -255,10 +254,47 @@ the same configuration parameters (if applicable). This is because the scheduler
 only has one pending pods queue.
 {{< /note >}}
 
+## Scheduler configuration migrations
+
+{{< tabs name="tab_with_md" >}}
+{{% tab name="v1beta2 → v1beta3" %}}
+* With the v1beta3 configuration version, you will have increased weights for the following user specifiable scoring plugins.
+  - TaintTolerations to 3
+  - NodeAffinity to 2
+  - InterPodAffinity to 2
+  - Won't have HealthzBindAddress, MetricsBindAddress fields 
```

Comment:
should be `*` instead of `-`. Also give the alternative, which I believe is the command line flags `--bind-address` and `--secure-port`, but please double check